public/index.html

```diff
@@ -27,7 +27,14 @@ <h2 class="text-2xl text-green-50 italic">Frontend Developer-Writer</h2>
         <div class="absolute top-0 right-0 mt-1 mr-1">
             <img src="images/2202-2020-101115569136863779874-01 (1)-01-01.jpeg.jpg" alt="Adhithya Rajan's Photo" class="max-w-md rounded-full shadow-lg">
         </div>
-        <main class="bg-zinc-900 text-pink-700">
+        <main class="bg-zinc-900 text-pink-700 relative flex">
+            <div class="w-1/4 p-40">
+                <div class="mt-1 mr-1">
+                    <a href="https://docs.google.com/document/d/1S0JpUkxcW8MaNZTf6bv-dHpvvfPhD0kv/edit?usp=drivesdk&ouid=110576272185341512116&rtpof=true&sd=true">
+                        <img src="images/upload_cv-512.webp" alt="Adhithya Rajan's Resume" class="max-w-md shadow-lg border-4 border-pink-700 hover:bg-white">
```

Comment:
If you can find an svg of the same icon it would be better

Reply:
@ajayesivan I've added the only one cv icon i got in svg format. But i think previous one was good. Please check PR and let me know what you think.

Reply:
The current one is okay. Don't worry too much about the UI & UX atm. We will definitely help you improve it down the line.